azure/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -73,7 +73,11 @@ resource "azurerm_virtual_machine_scale_set" "workers" {
 
   # eviction policy may only be set when priority is Low
   priority        = "${var.priority}"
-  eviction_policy = "${var.priority == "Low" ? "Delete" : null}"
+  # It is not possible to assign an optional argument in terraform 0.11,
+  # (https://github.com/hashicorp/terraform/issues/17968)
+  # however azurerm provider v1.35 ignores the value if the priority
+  # is not low. https://github.com/terraform-providers/terraform-provider-azurerm/blob/3b0ba54174ed56d2082228349b1bc19c38b75eca/azurerm/resource_arm_virtual_machine_scale_set.go#L866
```

Comment:
> Uff, you had to look at the code to catch this? Awesome work!
>

Yes, I had fun looking at the code!

> This can be set to "null" on terraform 0.12, right?

Right, in-fact that's the problem, it was updated to use `null` but we are still running Terrraform 0.11. Actually the PR for Terraform 0.12 will have to revert this change as the current solution is good for it. Since I'm the one working on the other PR I'll avoid making a comment :), I hope TF0.12 will be merged soon too.

Reply:
:ok_hand: